Cargo.toml

```diff
@@ -11,13 +11,13 @@ exclude = ["decodecorpus_files/*", "dict_tests/*", "fuzz_decodecorpus/*"]
 readme = "Readme.md"
 
 [dependencies]
-byteorder = "1.3.4"
-twox-hash = { version = "1.6.0", default-features = false }
+byteorder = "1.4.3"
+twox-hash = { version = "1.6.2", default-features = false }
```

Comment:
What's the reason for bumping those? `cargo update` is already able to update them.

Reply:
Oh really? 😕

Reply:
Let's see what @KillingSpark thinks